### PR TITLE
EWPP-930: Drupal 9 compatibility fixes.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

**Review by drishu — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -6,21 +6,21 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7",
-        "php": ">=7.2",
-        "nesbot/carbon": "^2.29"
+        "drupal/core": "^8.9 || ^9",
+        "php": ">=7.3",
+        "nesbot/carbon": "^2.47"
```
Comment (upchuk) [inline composer.json:11]: Why are you increasing this (and all other deps that don't have to be increased)?

**Review by upchuk — CHANGES_REQUESTED** [review]
You are missing changes in the phpunit.xml.dist file, grump (status A paraitre should be orderable)

Please check the other tickets and lets bring these components up to date.

File: oe_time_caching.info.yml [hunk 2]
```diff
@@ -3,4 +3,4 @@ description: OpenEuropa Time Caching provides time-based cache tags and invalida
 package: OpenEuropa
```
Comment (yenyasinn) [inline oe_time_caching.info.yml:3]: Would be good to wrap module's name and description by single quote as in https://github.com/openeuropa/oe_webtools/pull/156/files#diff-c61f63280f6689204a18fcb67a674514314c749361707af3cd04340c1af1fadc.

File: composer.json [hunk 3]
```diff
@@ -6,21 +6,21 @@
...
     "require-dev": {
         "composer/installers": "~1.5",
-        "drupal/core-composer-scaffold": "^8.8",
-        "drupal/config_devel": "~1.2",
+        "drupal/core-composer-scaffold": "^8.9 || ^9",
+        "drupal/config_devel": "^1.8",
         "drupal/drupal-extension": "~4.0",
-        "drush/drush": "~9.0@stable",
+        "drush/drush": "~10.3",
         "guzzlehttp/guzzle": "~6.3",
```
Comment (yenyasinn) [inline composer.json:19]:
```suggestion
```
We don't need `"guzzlehttp/guzzle": "~6.3",` because it is already the dependency of the Drupal core

**Review by yenyasinn — COMMENTED** [review]
Behat tests aren't used here but in any case please wrap strings by quote as in https://github.com/openeuropa/oe_multilingual/pull/121/files#diff-50059421b2806d799959ec59ad6bfa4c6bb1156415eb804f1bbf62c77cae6b90

**Review by yenyasinn — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Pull request merged**